packages/uikit/src/widgets/Swap/CurrencyInputPanel.tsx

```diff
@@ -44,7 +45,7 @@ export function CurrencyInputPanel({
           className={inputContainerVariants({
             hasZapStyle: !!zapStyle,
             showBridgeWarning: !!showBridgeWarning,
-            error,
+            error: Boolean(error),
```

Comment:
can we make a variable so no need two place show  `Boolean(error)`
exp: 
`const isError = Boolean(error)`